doc/source/examples/02_many_model_training/many_model_training.py

```diff
@@ -0,0 +1,57 @@
+# For reading from cloud storage paths.
+from smart_open import smart_open
+import pandas as pd
+
+import ray
+from ray import air, tune
+from ray.air import session
+
+ray.init()
+
+
+def trainable_func(config: dict):
+    data = pd.read_csv(smart_open(config["file_path"], "r"))
+
+    # Train your model here.
+    from sklearn.linear_model import LinearRegression
+
+    X, y = data[["id4", "id5"]], data["v3"]
+    reg = LinearRegression().fit(X, y)
+    score = reg.score(X, y)
+
+    # Report a single dict of the model and stats, etc.
+    session.report(
+        metrics={
+            "coef": reg.coef_,
+            "intercept": reg.intercept_,
+            "customer_id": data["customer_id"][0],
+            "score": score,
+        }
+    )
+
+
+# Tune is designed for up to thousands of trials.
+num_nodes = len(ray.nodes())
```

Comment:
Remove num nodes.

Reply:
I'm using the number of nodes to distinguish between the hello world / larger scale examples. Is this too implicit - and you'd prefer providing a flag when running the script instead? Or is it becuase `ray.nodes()` is not something we should show the user?